examples/apps/generic_client/proxy_mode_generic_client.py

```diff
@@ -0,0 +1,377 @@
+#! /usr/bin/env python3
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import json
+import argparse
+import logging
+import secrets
+import asyncio
+
+import config.config as pconfig
+import utility.logger as plogger
+import utility.hex_utils as hex_utils
+import avalon_crypto_utils.crypto_utility as crypto_utility
+from avalon_sdk.worker.worker_details import WorkerType, WorkerStatus
+import avalon_sdk.worker.worker_details as worker_details
+from avalon_sdk.work_order.work_order_params import WorkOrderParams
+from error_code.error_status import WorkOrderStatus
+import avalon_crypto_utils.signature as signature
+from error_code.error_status import SignatureStatus
+from avalon_sdk.connector.blockchains.fabric.fabric_worker_registry \
+    import FabricWorkerRegistryImpl
+from avalon_sdk.connector.blockchains.fabric.fabric_work_order \
+    import FabricWorkOrderImpl
+from avalon_sdk.connector.blockchains.ethereum.ethereum_worker_registry \
+    import EthereumWorkerRegistryImpl
+from avalon_sdk.connector.blockchains.ethereum.ethereum_work_order \
+    import EthereumWorkOrderProxyImpl
+from avalon_sdk.connector.blockchains.common.contract_response \
+    import ContractResponse
+
+TCF_HOME = os.environ.get("TCF_HOME", "../../")
```

Comment:
TCF_HOME is 3 directories back from directory of this file. Please change to "../../../"